### PR TITLE
🪄 Fix discussions

### PR DESCRIPTION
**Pull request by lucas-luchack**
I have done all the following:

- [x] added the feature
- [x] ran mypy, black, and unit tests

### CODE REVIEW

File: pronotepy/dataClasses.py [hunk 1]
```diff
@@ -123,6 +123,12 @@ def date_parse(formatted_date: str) -> datetime.date:
         elif re.match(r"\d{2}/\d{2}", formatted_date):
             formatted_date += f"/{datetime.date.today().year}"
             return datetime.datetime.strptime(formatted_date, "%d/%m/%Y").date()
+        elif re.match(r"\d{2}\d{2}$", formatted_date):
+            date = datetime.date.today()
+            hours = int(formatted_date[:2])
+            minutes = int(formatted_date[2:])
+            formatted_date = f"{date:%d}/{date:%m}/{date.year} {hours}h{minutes}"
```
Comment (bain3) [inline pronotepy/dataClasses.py:130]: Are you sure this is correct? `hours` and `minutes` are `int`s, so by default, there are no leading zeroes. AFAIK `%Hh%M` only matches when it has leading zeroes.

Comment (bain3) [reply pronotepy/dataClasses.py:130]: Never mind, tried it, and it works.

File: pronotepy/dataClasses.py [hunk 2]
```diff
@@ -123,6 +123,12 @@ def date_parse(formatted_date: str) -> datetime.date:
         elif re.match(r"\d{2}/\d{2}", formatted_date):
             formatted_date += f"/{datetime.date.today().year}"
             return datetime.datetime.strptime(formatted_date, "%d/%m/%Y").date()
+        elif re.match(r"\d{2}\d{2}$", formatted_date):
```
[the remark this answers is not in the document]
Comment (lucas-luchack) [reply pronotepy/dataClasses.py:126]: In discussion tab

[the remark this answers is not in the document]
Comment (lucas-luchack) [reply pronotepy/dataClasses.py:126]: image
image

[the remark this answers is not in the document]
Comment (bain3) [reply pronotepy/dataClasses.py:126]: Oh yeah, right, I think a better solution would be to use `msg["date"]["V"]`, and parse that, the format looks more stable.

message json in devtools

[the remark this answers is not in the document]
Comment (lucas-luchack) [reply pronotepy/dataClasses.py:126]: the problem is this is not the creation date of the discussion but the last message...

[the remark this answers is not in the document]
Comment (bain3) [reply pronotepy/dataClasses.py:126]: ah, okay then

**Pull request merged**